src/status_im2/common/scroll_page/view.cljs

```diff
@@ -64,26 +64,15 @@
       (if top-nav
         [rn/view {:style {:margin-top (if platform/ios? 44 0)}}
          top-nav]
-        [rn/view {:style {:margin-top 44}}
-         [quo/page-nav
-          (merge
-           {:horizontal-description? true
-            :one-icon-align-left?    true
-            :align-mid?              false
-            :page-nav-color          :transparent
-            :mid-section             {:type            :text-with-description
-                                      :main-text       nil
-                                      :description-img nil}
-            :right-section-buttons   (if (= 1 reanimated/get-shared-value opacity-animation)
-                                       (assoc page-nav :icon-background :blur)
-                                       page-nav)}
-           (when navigate-back?
-             {:left-section {:icon            :i/close
-                             :type            :grey
-                             :icon-background (if (= 1 reanimated/get-shared-value opacity-animation)
-                                                :blur
-                                                :photo)
-                             :on-press        #(rf/dispatch [:navigate-back])}}))]])
+        [quo/page-nav
+         (cond-> {:margin-top 44
```

Comment:
i know that was before here, but any thoughts why `:margin-top 44` ?

Reply:
Pretty sure I initially put this here :/ it was to handle the "safe-area" space at the top, i.e this page needs to push up beyond the iphone status bar for the scrolling effect in the designs. I think it should be just swapped with safe area usage as we have since done on other page. 
Should be handled beyond this pr ofc because I think there are some other pieces that will also be affected if we just change this 👍